src/services/fileStorage.tsx

```diff
@@ -0,0 +1,45 @@
+/**********************************
+  
+ TODO: Will be moved to backend
+
+**********************************/
+
+import AWS from 'aws-sdk'
+
+//check clickup for the credentials
+const bucketName = '';
+const accessKeyId =  '';
+const secretAccessKey = '';
+const region = '';
+
+AWS.config.update({
+  accessKeyId: accessKeyId,
+  secretAccessKey: secretAccessKey,
+  region: region
+})
+
+const s3 = new AWS.S3();
+
+
+export const uploadFile = (fileName: string, filePath: string) => { 
```

Comment:
rename the `filePath` param to `body` or `base64`

Reply:
Got it. Will update this, thanks @whamester.